scripts/golden_diff_notify.sh

```diff
@@ -25,8 +27,7 @@ apt-get install -y gawk
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 ROOT="$(dirname "$DIR")"
 
-
-echo "$BASE_BT_VERSION" | tee "$ROOT/deploy/storage/bigtable.version"
+echo "$BASE_BT_VERSION" | tee "$ROOT/deploy/storage/bigtable_import_groups.version"
```

Comment:
$BASE_BT_VERSION is the full list of import groups which is produced in the cloudbuild.yaml file below.

Reply:
Ah, I see. (super nit: might be worth renaming the variable for readability... when/if you touch this next)

Also, it will be interesting to see if '\n' gets handled right...

Reply:
Yea, updated name in https://github.com/datacommonsorg/mixer/pull/785/files